### ISSUE TITLE
Fails if display environment variable is not set

### ISSUE THREAD

**Issue by dylanfprice** [alzadude/ansible-firefox-addon#1]
On my machine, $display is empty (instead it is $DISPLAY) which causes [ensure_profile](https://github.com/alzadude/ansible-firefox-addon/blob/master/library/firefox_addon#L18) to interpret "addon_profile" as the display arg, leading to this error message:

```
{ "failed": true, "msg": "couldn't determine profile path", "stdout": "Error: cannot open display: addon_profile" }
```

I am able to work around this by setting display equal to DISPLAY like so:

``` yaml
- firefox_addon:
    url: https://addons.mozilla.org/en-US/firefox/addon/ublock-origin/
    state: present
  environment:
    display: '{{ ansible_env.DISPLAY }}' 
```

Thanks for this role!

**Comment by dylanfprice** [comment]
Also FYI I am on Ubuntu 14.04.4 LTS

**Comment by alza-bitz** [comment]
Hi there,

Thanks for the report. I just pushed a fix (and a test) so you shouldn't need the workaround anymore.

TBH the 'display' argument for the module is hidden/undocumented because it's only really needed for getting the tests to run, and this is because I couldn't find a way to make it work using the DISPLAY environment variable instead, which would simplify the code. For reasons I don't fully understand, even if I set a DISPLAY environment variable in the test container which I can see when I connect with SSH, when I invoke 'ansible -m <my-module>' to connect, the DISPLAY environment is not present..

Thanks
